src/commands/me.rs

```diff
@@ -0,0 +1,47 @@
+use std::borrow::Cow;
+
+use matrix_sdk::ruma::events::{
+    message::MessageEventContent,
+    room::message::{NoticeMessageEventContent, RoomMessageEventContent},
+};
+use weechat::{
+    buffer::Buffer,
+    hooks::{CommandRun, CommandRunCallback},
+    ReturnCode, Weechat,
+};
+
+use crate::Servers;
+
+pub struct MeCommand {
+    servers: Servers,
+}
+
+impl MeCommand {
+    pub fn create(servers: &Servers) -> Result<CommandRun, ()> {
+        CommandRun::new(
+            "/me",
+            MeCommand {
+                servers: servers.clone(),
+            },
+        )
+    }
+}
+
+impl CommandRunCallback for MeCommand {
+    fn callback(
+        &mut self,
+        _: &Weechat,
+        buffer: &Buffer,
+        cmd: Cow<str>,
+    ) -> ReturnCode {
+        if let Some(room) = self.servers.find_room(buffer) {
+            self.servers.runtime().block_on(room.send_message(
+                RoomMessageEventContent::emote_plain(
+                    cmd.strip_prefix("/me ").map(|s| s.to_string()).unwrap(),
```

Comment:
Could also use split args and drop first element, no idea which is better

Reply:
I marginally like the `strip_prefix` more, then the rest could be parsed by `clap`.

Obviously that's not needed here.